package/entrypoint.sh

```diff
@@ -35,7 +35,9 @@ if [ -e /var/lib/rancher/k3s/server/db/etcd ]; then
   k3s server --cluster-init --cluster-reset &> ./k3s-cluster-reset.log
   K3S_CR_CODE=$?
   if [ "${K3S_CR_CODE}" -ne 0 ]; then
-    echo "ERROR:" && cat ./k3s-cluster-reset.log
+    echo "ERROR while running 'k3s server --cluster-init --cluster-reset', exit code ${K3S_CR_CODE}"
+    echo "Last 50 lines of k3s.log:" && tail -n 50 ./k3s.log
```

Comment:
Is there a case where you found the `k3s.log` useful for debugging a failing `k3s server --cluster-init --cluster-reset`?  `k3s.log` should not be mutated unless Rancher is successful in starting, which won't happen if the `k3s server --cluster-init --cluster-reset` fails. See: https://github.com/rancher/norman/blob/96b95e3475e0dd39b49bf27082c3f9f440b336e9/pkg/kwrapper/k8s/k3s_linux.go#L42

I have a feeling that these 50 lines will just end up muddying the waters as if this command fails, nothing will have changed in the `k3s.log`. 

If the goal is to get Rancher to spit out embedded `k3s` logs if it dies, then we should probably put this in norman IMO and catch corresponding signals that indicate we should dump logs. Bonus points if we can use a cancelcontext or something that cleans up the `K3s` process spawned by Rancher/norman.

Reply:
It's not specifically about debugging a failed `cluster-reset`, it's about getting more context about failing to start which could be after a successful `cluster-reset` (see https://github.com/rancher/rancher/issues/40598)

There have been plenty of examples in this repository where this file was requested to debug which is missing context for the user and for us as we can just display it directly. If you don't agree, we can just close it, my opinion is that this is an easy time saver.